docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -97,18 +115,44 @@ cluster. This can help if you encounter a bug which renders the user interface p
 It might also help if you have issues with connecting to an active cluster that don't happen in the
 Web UI.
 
+<Tabs>
+<TabItem label="macOS">
 To force the app to log you out of all clusters, close the app and remove the `~/Library/Application
-Support/Teleport Connect/tsh` folder. Removing the file under `~/Library/Application
+Support/Teleport Connect/tsh` folder. Removing the file `~/Library/Application
 Support/Teleport Connect/app_state.json` will clear all remembered tabs and connections.
+</TabItem>
+<TabItem label="Linux">
+To force the app to log you out of all clusters, close the app and remove the `~/.config/Teleport
+Connect/tsh` folder. Removing the file `/.config/Teleport Connect/app_state.json` will clear
+all remembered tabs and connections.
+</TabItem>
+<TabItem label="Windows">
+To force the app to log you out of all clusters, close the app and remove the
+`C:\Users\yourUser\AppData\Roaming\Teleport Connect\tsh` folder. Removing the file
+`C:\Users\yourUser\AppData\Roaming\Teleport Connect\app_state.json` will clear all remembered tabs
+and connections.
+</TabItem>
+</Tabs>
 
 ### Submitting an issue
 
 To submit an issue, click the Submit Feedback button at the bottom right (the speech bubble symbol)
 and follow the *Submit a Bug* link.
 
-Be sure to attach logs which can be found under `~/Library/Application Support/Teleport
-Connect/logs`. The version of the app can be found in the app menu under the About Teleport Connect
-menu item.
+<Tabs>
+<TabItem label="macOS">
+Be sure to attach logs which can be found under `~/Library/Application Support/Teleport Connect/logs`.
+The version of the app can be found in the app menu under the About Teleport Connect menu item.
+</TabItem>
+<TabItem label="Linux">
+Be sure to attach logs which can be found under `~/.config/Teleport Connect/logs`. The app version
+can be found by pressing `Alt` to access the app menu -> Help -> About Teleport Connect
+</TabItem>
+<TabItem label="Windows">
+Be sure to attach logs which can be found under `C:\Users\yourUser\AppData\Roaming\Teleport Connect\logs`.
```

Comment:
Based on my previous comment, technically we could do this here:

```suggestion
Be sure to attach logs which can be found under `%AppData%\Teleport Connect\logs`.
```

But I think a regular user wouldn't know how to navigate to `%AppData%` so perhaps it's better to leave it as it is. Or maybe we should just use it and don't assume stuff unnecessarily, what do you think?

OTOH, at one of my previous jobs we just added a button to the Help menu which opened the logs folder.

Reply:
One problem is that `AppData` is a hidden folder and those are (I think) hidden by default on Windows. 🤔 So if they're just following the path by opening each subsequent folder in the file explorer they might run into a problem anyway.

Reply:
I think the solution here is in the Stack Exchange link you posted on the other comment; use the `%UserName%` var to show the path while also being paste-able. We can also reference hidden files:

```suggestion
Be sure to attach logs which can be found under `C:\Users\%UserName%\AppData\Roaming\Teleport Connect\logs`.
You may need to adjust File Explorer to[view hidden files and folders](https://support.microsoft.com/en-us/search?query=how%20to%20view%20hidden%20files%20in%20windows%2010).
```

Reply:
Oh, that's smart!